### PR TITLE
[GFA1] improve definition of C lines

### PR DESCRIPTION
**Pull request by ggonnella**
This addresses issue #57

### CODE REVIEW

File: GFA1.md [hunk 1]
```diff
@@ -132,17 +132,26 @@ The Overlap field is optional and can be `*`, meaning that the CIGAR string is n
 
 # `C` Containment line
 
-A containment line represents an overlap between two segments where one is contained in the other.
+A containment line represents an overlap between two segments where one (the `From` segment)
+is contained in the other (the `To` segment). The `Pos` field stores the leftmost
+position of the contained segment in the container segment in its forward orientation
+(i.e. before this is oriented according to the `FromOrient` sign).
+
+E.g. the following line describes a containment of segment 2 in the reverse complement of segment 1,
```
Comment (sjackman) [inline GFA1.md]: Please replace `e.g.` with a Markdown heading `## Example`

> describes a containment

describes the containment

**Comment by sjackman** [comment]
Thanks for this clarification, Giorgio. It's needed.

File: GFA1.md (hunk 1 above)
Comment (ggonnella) [inline GFA1.md]: Agree

**Comment by ggonnella** [comment]
@sjackman I agree with changing the name of the fields too (I added a commit to the PR, which implements it). The names "from" and "to" were probably just inherited from L, but do not really make sense in C lines anyway.

**Comment by sjackman** [comment]
Looks good to me. Thanks, Giorgio. This PR needs one more :+1: to be merged.

**Comment by sjackman** [comment]
@eshcherbin Care to vote on this PR, Egor?

**Comment by eshcherbin** [comment]
:+1: 
Yes, I think these changes should be made, the new names make more sense than the old ones.

**Comment by sjackman** [comment]
Thanks! I'll wait until Tuesday 2017-07-04 to merge this PR, to wait for more votes. Please comment here if I forget.

**Comment by sjackman** [comment]
This PR has merge conflicts.
@ggonnella Could you please rebase it onto the `master` branch?

**Comment by ggonnella** [comment]
Done, see #65

**Comment by sjackman** [comment]
You can push to the same branch to update a PR.
Superseded by https://github.com/GFA-spec/GFA-spec/pull/65

**Pull request closed without merging**